pkg/agent/manager/manager.go

```diff
@@ -22,9 +22,16 @@ import (
 	"github.com/spiffe/spire/pkg/common/rotationutil"
 	"github.com/spiffe/spire/pkg/common/telemetry"
 	"github.com/spiffe/spire/pkg/common/util"
+	"github.com/spiffe/spire/pkg/server/api/limits"
 	"github.com/spiffe/spire/proto/spire/common"
 )
 
+const (
+	maxSVIDSyncInterval = 4 * time.Minute
+	// for sync interval of 5 sec this will result in max of 4 mins of backoff
+	synchronizeMaxIntervalMultiple = 48
```

Comment:
if a user set the interval to be 1m this means we may wait 48m in the worst scenario, may we add a max value?

Reply:
should such a max be configurable, or should that be an opinionated sane hard-coded value?

Reply:
Hard coded the max value to 8 mins